js/model3D/interactive/Model3DUpload.svelte

```diff
@@ -88,7 +88,14 @@
 	import * as BABYLON_LOADERS from "babylonjs-loaders";
 	import type { I18nFormatter } from "js/utils/src";
 
-	BABYLON_LOADERS.OBJFileLoader.IMPORT_VERTEX_COLORS = true;
+	$: {
+		if (
+			BABYLON_LOADERS.OBJFileLoader != undefined &&
+			!BABYLON_LOADERS.OBJFileLoader.IMPORT_VERTEX_COLORS
+		) {
+			BABYLON_LOADERS.OBJFileLoader.IMPORT_VERTEX_COLORS = true;
+		}
+	}
```

Comment:
Hmm, I'm not sure this will ever rerun. I also don't actually know what this does.

@hannahblair @dawoodkhan82 any ideas?

Reply:
This imports the colors that some `.obj` files have. I think `BABYLON_LOADERS.OBJFileLoader.IMPORT_VERTEX_COLORS = true;` could go in `on_mount()`.

Reply:
I tried adding it to `onMount` but I still get the same error 😅

```js
Model3DUpload.svelte:96 Uncaught (in promise) TypeError: Cannot set properties of undefined (setting 'IMPORT_VERTEX_COLORS')
    at instance (Model3DUpload.svelte:96:64)
    at init (svelte.js:3237:5)
    at new Model3DUpload (Model3DUpload.svelte:116:28)
    at createProxiedComponent (svelte-hooks.js:341:9)
    at new ProxyComponent (proxy.js:242:7)
    at new Proxy<Model3DUpload> (proxy.js:349:11)
    at Array.create_default_slot_2 (Index.svelte:98:22)
    at create_slot (svelte.js:165:23)
    at create_dynamic_element (Block.svelte:35:23)
    at create_fragment (Block.svelte:22:11)
```

Reply:
I can't seem to reproduce this issue. Are you just running the `model3D` demo on frontend dev mode?

Reply:
Running on custom components dev mode. So the following was not working for me:

```bash
gradio cc create MyModel3D --template Model3D --install
cd mymodel3d
gradio cc dev
```